.travis.yml

```diff
@@ -1,12 +1,12 @@
 language: python
 dist: xenial
 python:
-    - 2.7
+    - 3.5
```

Comment:
Python 3.5 is EOL in September this year I think, please use a more recent version.

Reply:
So actually I'd recommend going for a travis matrix build ... I'll set this up and include 3.5+

Reply:
I tried with a travis file, but for some reason travis doesn't want to show me anything ... and doesn't let me activate it on my clone ... hmm.

Reply:
Ok, going back to a simple 3.6 build works. Fair enough for now.